app/commands/github/organization_member/remove.rb

```diff
@@ -0,0 +1,13 @@
+module Github
+  class OrganizationMember
+    class Remove
```

Comment:
I don't think this is used anywhere yet.

Reply:
Not yet, but it will be once I modify the controllers to use the commands.